src/libs_3rdparty/QSpec/src/primitives/GTWidget.cpp

```diff
@@ -226,18 +226,16 @@ QList<QLabel *> GTWidget::findLabelByText(GUITestOpStatus &os,
                                           QWidget *parentWidget,
                                           const GTGlobals::FindOptions &options) {
     QList<QLabel *> resultLabelList;
-    for (int time = 0; time < GT_OP_WAIT_MILLIS; time += GT_OP_CHECK_MILLIS) {
-        GTGlobals::sleep(time > 0 ? GT_OP_CHECK_MILLIS && resultLabelList.isEmpty() : 0);
+    for (int time = 0; time < GT_OP_WAIT_MILLIS && resultLabelList.isEmpty(); time += GT_OP_CHECK_MILLIS) {
```

Comment:
The same about lines. It would be so much better, if you make just the separate line like this:
`CHECK_CONTINUE(resultLabelList.isEmpty());`

Reply:
There are about 20 time-out waiters like this in our code. Almost all of them use this pattern.